### ISSUE TITLE
Error in the first frame when trying to train the wan 2.2 i2v dual noise

### ISSUE THREAD

**Issue by hankunbo** [kohya-ss/musubi-tuner#744]
Error in the first frame when trying to Inference the wan 2.2 i2v dual noise,
my first frame is 

![Image](https://github.com/user-attachments/assets/8931cce5-5d17-4430-b42f-7ecc0baa76fe)

but it come out to be quite blur in the first frame in the video ,
here is the first frame of the video:

<img width="1020" height="1012" alt="Image" src="https://github.com/user-attachments/assets/49a2758a-1ea0-4414-a22b-29e6041309a7" />

this is my training Instruction：
accelerate launch  \
--num_cpu_threads_per_process 1 \
--mixed_precision fp16 \
src/musubi_tuner/wan_train_network.py \
--task i2v-A14B \
--dit models/wan2.2/diffusion_models/wan2.2_i2v_low_noise_14B_fp16.safetensors \
--dit_high_noise models/wan2.2/diffusion_models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
--dataset_config datasets/wan22_dog.toml \
--sdpa \
--mixed_precision fp16 \
--fp8_base \
--optimizer_type adamw8bit \
--learning_rate 2e-4 \
--gradient_checkpointing \
--max_data_loader_n_workers 1 \
--persistent_data_loader_workers \
--network_module networks.lora_wan \
--network_dim 32 \
--timestep_sampling shift \
--discrete_flow_shift 5.0 \
--max_train_epochs 300 \
--save_every_n_epochs 10 \
--seed 42 \
--output_dir output2 \
--output_name big_headed_dog_low3 \
--blocks_to_swap 35 \
--optimizer_args "weight_decay=0.01" \
--network_args "loraplus_lr_ratio=16"  "loraplus_text_encoder_lr_ratio=4" \
--preserve_distribution_shape

and this is my Inference Instruction:
python src/musubi_tuner/wan_generate_video.py \
--task i2v-A14B \
--video_size 832 480 \
--video_length 21 \
--infer_steps 20 \
--fp8 \
--fp8_scaled \
--fp8_t5 \
--prompt "big-headed dog is running." \
--save_path output/save \
--output_type both \
--dit models/wan2.2/diffusion_models/wan2.2_i2v_low_noise_14B_fp16.safetensors \
--dit_high_noise models/wan2.2/diffusion_models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
--vae models/wan2.2/vae/wan_2.1_vae.safetensors \
--t5 models/wan2.2/text_encoders/models_t5_umt5-xxl-enc-bf16.pth \
--attn_mode sdpa \
--image_path images/2.jpg \
--lora_weight output2/big_headed_dog_low3.safetensors \
--lora_multiplier 1.0

I only meet this issus when training the dual noise,
after trying,it has nothing to be done with --fp8 --fp8_scaled --fp8_t5
